provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -472,20 +475,182 @@ tasks:
         echo 'WARNING: use the information above to configure your Keycloak deployment!'
 
   dataset-import:
-    desc: "Triggers the dataset creation task in the primary Keycloak cluster. It does not wait for completion."
+    internal: true
+    requires:
+      vars:
+        - KC_URL
+    vars:
+      USERS: "{{.USERS | default 100000}}"
+      USERS_PER_TX: "{{.USERS_PER_TX | default 256}}"
+      THREADS: "{{.THREADS | default 8}}"
+    cmds:
+      - curl -k "https://{{.KC_URL}}/realms/master/dataset/create-realms?realm-name=realm-0&count=1&threads-count={{.THREADS}}&users-per-realm={{.USERS}}&users-per-transaction={{.USERS_PER_TX}}"
+
+  dataset-import-route53:
+    desc: "Triggers the dataset creation task using the Route53 load balancer. It does not wait for completion."
     deps:
       - common:split
       - common:env
     requires:
       vars:
-        - ROSA_CLUSTER_NAME_1
+        - KC_CLIENT_URL
+    cmd:
+      task: dataset-import
+      vars:
+        KC_URL: "{{.KC_CLIENT_URL}}"
+
+  dataset-import-rosa-cluster:
+    desc: "Triggers the dataset creation task in the ROSA cluster route. It does not wait for completion."
+    deps:
+      - common:split
+      - common:env
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+        - KC_NAMESPACE_PREFIX
+    cmds:
+      - task: ispn:rosa-oc-login
+      - task: dataset-import
+        vars:
+          KC_URL:
+            sh: echo "keycloak-{{.KC_NAMESPACE_PREFIX}}keycloak.$(cat {{.ISPN_DIR}}/.task/kubecfg/ocp-prefix-{{.ROSA_CLUSTER_NAME}})"
+
+  ec2-create:
+    desc: "Creates the EC2 workers"
```

Comment:
I think this is a good entry point, but we would still need to route back to the `ANSIBLE_DIR`'s `env.yml`. Would the expectation be control the configuration from there instead of the task env variables ?

```
# Create `env.yml` for the `aws_ec2.sh` and `benchmark.sh` scripts to pick up custom parameters.

# Overrides for AWS EC2. Defaults located in `roles/aws_ec2/defaults/main.yml`.
cluster_size: 3 
instance_type: t3.small
instance_volume_size: 30

# Overrides for Keycloak Benchmark. Defaults located in `roles/benchmark/defaults/main.yml`.

# This will download this version from the GitHub releases
# kcb_version: 0.9-SNAPSHOT

# This will use a locally built version
kcb_zip: ../benchmark/target/keycloak-benchmark-0.10-SNAPSHOT.zip

kcb_heap_size: 1G

```

Reply:
> Would the expectation be control the configuration from there instead of the task env variables ?

do you mean the other way around, Ignoring `env.yml` and using task variables? This PR it uses the configuration from `env.yml`.

Reply:
right, thats what I meant :) sorry it came across meaning opposite. @pruivo

Reply:
I wasn't planning to do it.